src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -63,6 +63,8 @@
         CHECK_TO_ANCESTOR_NODES_MAP.put("FinalLocalVariable",
                 Arrays.asList(TokenTypes.METHOD_DEF,
                 TokenTypes.VARIABLE_DEF, TokenTypes.CTOR_DEF));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("InnerTypeLast",
+                Arrays.asList(TokenTypes.CLASS_DEF));
```

Comment:
Inner types are allowed not only in classes. Please add `INTERFACE_DEF`, `RECORD_DEF` and `ENUM_DEF` (even though enum is not expected in the check https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java#L110)

Reply:
ok, done